examples/lead-generation/src/cli/lead-generation.cli.ts

```diff
@@ -0,0 +1,72 @@
+import fs from "node:fs";
+import { leadGenerationGraph } from "@/graph/graph";
+import { confirm, input } from "@inquirer/prompts";
+import { getLogger } from "@local/utils";
+
+const log = getLogger();
+
+const main = async () => {
+  log.info("Welcome to the Lead Generation Recipe CLI!");
+
+  const urls: string[] = [];
+
+  // Collect URls
+  while (true) {
+    const url = await input({
+      message: "Enter a URL (or press Enter with empty input to finish):",
```

Comment:
For cli, I would add this suggestion to the URL, otherwise, they can't test it quickly.
```suggestion
      message: "Enter a URL (i.e. https://www.findapsychologist.org/cities/psychologists-in-san-francisco/ or press Enter with empty input to finish):",
```